# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../luaffi/lua-lib.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- luaffi-0.9.0/conf.mak.vanilla	2008-08-23 11:11:39.000000000 +0200
+++ luaffi-0.9.0/conf.mak	2008-08-23 11:11:52.000000000 +0200
@@ -18,7 +18,7 @@
 CFLAGS		= -I. $(LUAINCS) $(WIN32FLAG) $(WIN32CFLAGS)
 CXXFLAGS	= -I. $(LUAINCS) $(WIN32FLAG) $(WIN32CFLAGS)
 
-LDFLAGS		= -lffi -llua5.1 -lm $(WIN32FLAG)
+LDFLAGS		= -lffi -llua -lm $(WIN32FLAG)
 
 RCFLAGS		= $(CFLAGS)
 RCXXFLAGS	= $(CXXFLAGS)
